.github/workflows/Test.yml

```
@@ -0,0 +1,28 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Maybe better to run two tests in parallel?
```suggestion
    runs-on: ubuntu-latest
    strategy:
      matrix:
        group:
          - Core
          - Downstream
```